admin_guide/pruning_resources.adoc

```diff
@@ -342,6 +392,58 @@ them from ever being pruned.
 xref:../dev_guide/managing_images.adoc#tag-naming[Learn more about _istag_
 naming.]
 
+[discrete]
+[[using-secure-connection-against-insecure-registry]]
+==== Using secure connection against insecure registry
+
+If you see similar message in the output of the `oadm prune images` command:
```

Comment:
If you see a message similar to the following in the output of the `oadm prune images` command, then your registry is not secured and the `oadm prune images` client will attempt to use a secure connection: